File: javascript/fizzbuzz/fizzbuzz.js
function fizzbuzz(max) {
  if (max < 1) { return; }
  for (let i = 1; i <= max; i++) {
    let m3 = !( i % 3 );
    let m5 = !( i % 5 );
    if (m3 && m5) {
      console.log("FizzBuzz");
    } else if (m3) {
      console.log("Fizz");
    } else if (m5) {
      console.log("Buzz");
    } else {
      console.log( i );
    }
  }
}

fizzbuzz( 20 );

File: python/fizzbuzz/fizzbuzz.py
n = 100
for x in range(1,n+1):
  xmod3 = x % 3 == 0
  xmod5 = x % 5 == 0
  if (xmod3 and xmod5):
    print 'FizzBuzz'
  elif (xmod3):
    print 'Fizz'
  elif (xmod5):
    print 'Buzz'
  else:
    print x
